apps/website/.vuepress/theme/styles/overrides.scss

```diff
@@ -289,6 +289,10 @@ body .no-clr-ui p {
   //margin-bottom: 0;
 }
 
+[cds-text~='body'] li {
+  color: var(--clr-p1-color);
```

Comment:
Update to use `--cds-global-color-gray-900`

Reply:
700 is the correct color.